reactos/drivers/usb/usbhub/pdo.c

```diff
@@ -146,6 +146,11 @@ IsValidPDO(
 
     ChildDeviceExtension = (PHUB_CHILDDEVICE_EXTENSION)DeviceObject->DeviceExtension;
     ASSERT(ChildDeviceExtension->Common.IsFDO == FALSE);
+
+    // This can happen when parent device was surprise removed.
```

Comment:
I don't think this happens right now, does it? I don't see a ParentDeviceObject = NULL assignment.